documentation/src/docs/deployment/run-local.md

```diff
@@ -9,31 +9,31 @@ You may obtain a copy of the License at
 -->
 # Running Pravega
 
-Running in local machine allows us to get started using Pravega very quickly. Most of the options uses the standalone mode which is suitable for most of the development and testing purposes.
+
+Running in local machine allows us to get started using Pravega very quickly. Running Pravega in Standalone mode is suitable for development and testing Pravega applications.
 
 The prerequisites for running in local machine is described below.
 
 ## Standalone Mode
 
 ### From Source
 
-The Pravega source code needs to be checked out to download the dependencies:
+The Pravega source code needs to be checked out:
 
 ```
 git clone https://github.com/pravega/pravega.git
 cd pravega
 ```
 
-As a next step compile Pravega and start the standalone deployment.
+Compile Pravega and start the standalone deployment:
 
 ```
 ./gradlew startStandalone
 ```
 
 ### From Installation Package
 
-Download the Pravega latest release from the [Github Releases](https://github.com/pravega/pravega/releases). The tarball or zip files can be used as they are identical. Instructions are provided for the tar files, but the same can be used for the zip file also.
-
+Download the Pravega latest release from the [GitHub Releases](https://github.com/pravega/pravega/releases). The tarball or zip files can be used as they are identical. Instructions are provided for the tar files, but the same can be used for the zip file also.
```

Comment:
"Download the latest release of Pravega..." sounds better to me. 

About the sentences: "The tarball or zip files can be used as they are identical. Instructions are provided for the tar files, but the same can be used for the zip file also."  
* I'd suggest using the active voice like the previous sentence which starts with "Download the latest release..."
* "The tarball or zip files can be used as they are identical" looks incomplete to me. Perhaps the word `either` is missing.
* The sentences sound be made simpler: "Extract either the tarball or the zip file to a ...."

Reply:
Accepted suggestion